mix.exs

```diff
@@ -1,29 +1,14 @@
-defmodule RebarHexDoc.MixProject do
+defmodule Rebar3ExDoc.MixProject do
   use Mix.Project
-
+  @ex_doc_version "0.30.5"
   def project do
     [
-      app: :rebar_ex_doc,
+      app: :rebar3_ex_doc,
       version: "0.2.20",
       elixir: "~> 1.13",
-      start_permanent: Mix.env() == :prod,
-      deps: deps(),
+      deps: [ex_doc: "~> #{@ex_doc_version}"],
```

Comment:
The user of the `~>` op here doesn't make sense and that's my bad 😄 We should pin to the exact version.

Reply:
3d5d00a